docs/javascript/api.md

```diff
@@ -1185,22 +1213,53 @@ window.lumapps.customize(({ onNavigation }) => {
 });
 ```
 
-`onNavigation` is a function that will be called on each navigation. It receives the following parameters:
-- `currentPage`: Id of the current page.
+`onNavigation` is a function that will be called on each navigation. **This function is now deprecated**. Please `on` function with the [Navigation event](./api#eventsnavigation) instead.
```

Comment:
Can we move this just below the `### on navigation` and add an `IMPORTANT` prefix in bold?

Reply:
Done as well